packages/@o3r/application/schemas/cms.schema.json

```
@@ -31,9 +31,36 @@
       "type": "string",
       "description": "The relative path to the functional contents metadata folder to import"
     },
+    "agateEnabled": {
+      "type": "boolean",
+      "description": "Specify if Agate is enabled or not in the preview"
+    },
+    "baseNpmName": {
+      "type": "string",
+      "description": "if agateEnabled is set to true, this is used to set the proper Agate form type in the preview"
+    },
+    "features": {
+      "type": "object",
+      "description": "Contains features to be enabled or disabled",
+      "properties": {
+        "themingCssOverrideEnabled ": {
+          "type": "boolean",
+          "description": "Enable CSS override in theming"
+        },
+        "themingCssVariablesEnabled ": {
+          "type": "boolean",
+          "description": "Enable CSS variables in theming"
+        }
+      }
+    },
     "migrationScriptFolder": {
       "type": "string",
       "description": "The relative path to the migration scripts"
+    },
+    "migrationScriptVersionDependency": {
+      "type": "string",
+      "description": "Name of the dependency to get the current blueprint version. If the version is not found, version in the package.json will be used",
```

```suggestion
      "description": "Name of the package dependency to get the current blueprint version from. If the dependency is not found, the field version in the package.json will be used instead",
```

(please notice that `version` is not a `package.json` **mandatory** field and may be `undefined` as well)

---

I propose: "Name of the package dependency to get the current blueprint version from. If the dependency is not found, the field version in the package.json will be used instead if defined. If not defined, no migration script will be executed."